connexion/decorators/coroutine_wrappers.py

```diff
@@ -13,23 +13,23 @@ def get_request_life_cycle_wrapper(function, api, mimetype):
     :rtype asyncio.coroutine
```

Comment:
There's a `yield from` on line 9. Though commented, we might as well update it.
You can see this in my [await](https://github.com/cognifloyd/connexion/commit/0ad493e89179642b06a2c7b8eb2a2bffca75f85c) branch.